spec/controllers/report_controller_spec.rb

```diff
@@ -1223,6 +1223,33 @@
       EvmSpecHelper.local_miq_server
     end
 
+    context "when generating reports" do
+      render_views
+
+      before :each do
+        stub_user(:features => :all)
+        @rpt = FactoryGirl.create(:miq_report)
```

Comment:
Please don't do this, use [`let` and/or `let!`](http://www.betterspecs.org/#let).